lib/configuration/client_configuration.js

```diff
@@ -46,9 +48,10 @@ function getDefaultDirectories() {
       }
     );
   } else {
-    Logger.getInstance().warn('Home directory of the user is not present');
+    Logger.getInstance().warn('Home directory of the user is not defined');
```

Comment:
Are you sure? AFAIK it might not be there at all

Reply:
I went for an unification with log message in line 37 and using word 'defined'. 
I believe the os.homedir() retrieves the defined homedir from the env variable, but it doesn't actually check, if it really exists in the filesystem.

<img width="216" alt="image" src="https://github.com/user-attachments/assets/a186dbf6-d0e6-4033-addd-881687946ee2">
In the attached picture I renamed homedir, but line`console.log(os.homedir());` still returned the old name.